.travis.yml

```diff
@@ -4,12 +4,22 @@ env:
   global:
     - SCAN_SKIP_BUILD=1
   matrix:
-    # Define the details for each test suite that should be run
-    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager"         SCAN_SDK="iphonesimulator"
+    # Test on iOS 10.2
+    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager"         SCAN_DEVICE="iPhone 5 (10.2)"
```

Comment:
Partly because fastlane defaults to the 5s if you don't specify (so I figured, that seems fine), and partly because it tends to be useful to test on older/smaller devices to catch edge cases that we might not see in our own testing.

On the other hand, I suppose that this reasoning should lead to the 5—but I also suspect that usage rates on that are low enough that it can be ignored. But then again, it's also the last 32-bit iPhone, so maybe it's worth testing there for that reason.

I guess I could go either way, on the 5 vs. 5s. What do you think?